configure.ac

```diff
@@ -227,6 +227,8 @@ AC_MSG_RESULT($enable_dap)
 if test "x$enable_remote_functionality" = xno ; then
   AC_MSG_WARN([All network access is disabled => DAP support disabled.])
   enable_dap=no
+  AC_MSG_WARN([All network access is disabled => NCZARR support disabled.])
```

Comment:
Leaving nczarr enabled despite the lack of network access is currently what we'd like to do, although there will be a bigger task ahead of us regarding a separation of 'local-nczarr functionality' vs. 'remote data access functionality', in the code.  It's why I've, for the upcoming release candidate, opted to leave `libcurl` as a dependency when nczarr is enabled. Maybe it's already handled, but I need to make sure nothing breaks if we compile nczarr support without linking libcurl.